recipes/smof/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "smof" %}
+{% set version = "2.14.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "965af9331f853939a074c51f8ae15037117feccece3cc0cec3fffb8cd0268c2b"
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - smof=smof.main:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python >3
+  run:
+    - python >3
+
+test:
+  imports:
+    - smof
+  commands:
+    - smof --help
+
+about:
+  home: "https://github.com/incertae-sedis/smof"
+  license: "MIT"
+  summary: "UNIX-style utilities for FASTA file exploration"
+  
+extra:
+  recipe-maintainers:
+#    - your_github_account
```

Comment:
Please either remove the extra section or put your github use name here.

Reply:
@daler Right, one sec